source/MRViewer/CMakeLists.txt

```diff
@@ -14,8 +14,10 @@ ENDIF() # MR_EMSCRIPTEN
 add_library(${PROJECT_NAME} SHARED ${SOURCES} ${HEADERS})
 
 IF(NOT MR_EMSCRIPTEN)
+pkg_check_modules(GTKMM REQUIRED gtkmm-3.0)
 if(GTKMM_FOUND)
-  target_include_directories( ${PROJECT_NAME} PUBLIC ${GTKMM_INCLUDE_DIRS})
+	include_directories(${GTKMM_INCLUDE_DIRS})
+  # target_include_directories( ${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:/usr/include/gtkmm-3.0>) # @todo
```

Comment:
why this?

Reply:
`${GTKMM_INCLUDE_DIRS}` consists of many semicolon-separated directories and `target_include_directories` goes mad with `$<BUILD_INTERFACE:${GTKMM_INCLUDE_DIRS}>`. However, `$<BUILD_INTERFACE:/usr/include/gtkmm-3.0>` is not enough to include all headers

Reply:
resolved: rollback to `target_include` works